health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -1,44 +1,360 @@
+                  className=""
+                  content="Tooltipkbjkjnjknk"
+                  description=""
+                  header=""
+                  style={{}}
+                /> */}
+              </>
+            )
+          )
+          break;
+        case "CREATION_DATE":
+          let epoch = row?.auditDetails?.createdTime;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)